src/main.cpp

```diff
@@ -4867,6 +4873,157 @@ void static ProcessGetData(CNode* pfrom)
     }
 }
 
+bool checkVerificationQueue(){
+    return verificationQueue.isEmpty();
+}
+
+void txVerificationThread(){
+    int i = 0;
+    while(!ShutdownRequested())
+    {
+        if (!checkVerificationQueue() && i < 100)
```

Comment:
Probably is more clear to test directly "verificationQueue.isEmpty()" and remove this method

Reply:
done https://github.com/andreysobol/zen/commit/0358cb2dc407f760210bb2157cb71ce1d5a2b9d8